channels/printer/client/cups/printer_cups.c

```diff
@@ -431,13 +429,6 @@ rdpPrinterDriver* cups_freerdp_printer_client_subsystem_entry(void)
 		uniq_cups_driver->driver.ReleaseRef = printer_cups_release_ref_driver;
 
 		uniq_cups_driver->id_sequence = 1;
-		uniq_cups_driver->defaultPrinter = _strdup(cupsGetDefault());
```

Comment:
While you moved this on the windows printer implementation to the `enum_printers` function, you failed to do the same for `cups`

Reply:
No, there is the same for cups, but it's very more concise. (No need to get the name of the default printer here)